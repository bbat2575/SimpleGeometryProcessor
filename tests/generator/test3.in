-N=20 -mindist=0 -rseed=1