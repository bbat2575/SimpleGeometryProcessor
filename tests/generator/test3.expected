-36.56, 34.74
26.38, -24.49
-0.46, -5.05
15.16, 28.87
-40.61, -47.17
33.58, -6.72
26.23, -49.79
-5.46, 22.15
-27.12, 44.53
40.14, -46.94
-47.46, 4.14
43.91, -11.88
-28.34, -7.79
-47.1, -27.83
-6.21, -0.42
-26.69, -26.91
-28.12, -4.04
-21.02, -47.85
33.76, 5.65
14.23, -31.41
